config/jobs/kubernetes/kops/kops-config.yaml

```diff
@@ -320,3 +320,34 @@ periodics:
       # docker-in-docker needs privileged mode
       securityContext:
         privileged: true
+
+postsubmits:
+  kubernetes/kops:
+  - name: kops-postsubmit
+    branches:
+    - master
+    skip_report: true
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181218-134e718ec-experimental
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/logs"
+        " "--timeout=60"
```

Comment:
Typo `=` should be `-`

Reply:
Thanks - saved me some head-scratching :-)